package.json

```diff
@@ -85,5 +85,8 @@
 		"ava": "nyc ava",
 		"test": "run-s build lint ava",
 		"precommit": "run-p test"
-	}
+	},
+	"resolutions": {
+		"natives": "1.1.3"
```

Comment:
@DanielRuf please, help me to understand more about this. Could you explain better?

Reply:
This is for yarn to use the natives-version instead of a newer / older one as there is an issue with old Node versions like Node 8 - I think it was the compatibility / some crash.